eventstore/test_views.py

```diff
@@ -387,6 +394,161 @@ def test_postbirth_other_passport_origin(self):
+                        "group_id": "group-id-of-context-message",
+                        "id": "message-id-of-context-message",
+                        "mentions": ["wa-id1", "wa-id2"],
+                    },
+                    "image": {
+                        "file": "absolute-filepath-on-coreapp",
+                        "id": "media-id",
+                        "link": "link-to-image-file",
+                        "mime_type": "media-mime-type",
+                        "sha256": "checksum",
+                        "caption": "image-caption",
+                    },
+                    "location": {
+                        "address": "1 Hacker Way, Menlo Park, CA, 94025",
+                        "name": "location-name",
+                    },
+                    "system": {"body": "system-message-content"},
+                    "text": {"body": "text-message-content"},
+                }
+            ]
+        }
+        response = self.client.post(
+            self.url,
+            data,
+            format="json",
+            HTTP_X_TURN_HOOK_SIGNATURE=self.generate_hmac_signature(data, "REPLACEME"),
+        )
+
+        self.assertEqual(response.status_code, status.HTTP_202_ACCEPTED)
+        [messages] = Messages.objects.all()
+        self.assertEqual(str(messages.contact_id), "sender-wa-id")
+        self.assertEqual(
+            messages.timestamp, datetime.datetime(2018, 2, 15, 11, 38, 20, tzinfo=UTC)
+        ),
+        self.assertEqual(messages.id, "9e12d04c-af25-40b6-aa4f-57c72e8e3f91"),
+        self.assertEqual(messages.type, "image"),
+        self.assertEqual(messages.message_direction, "I"),
+        self.assertEqual(
+            messages.data,
+            {
+                "context": {
+                    "from": "sender-wa-id-of-context-message",
+                    "group_id": "group-id-of-context-message",
+                    "id": "message-id-of-context-message",
+                    "mentions": ["wa-id1", "wa-id2"],
+                },
+                "audio": None,
+                "document": None,
+                "errors": None,
+                "video": None,
+                "voice": None,
+                "image": {
+                    "file": "absolute-filepath-on-coreapp",
+                    "id": "media-id",
+                    "link": "link-to-image-file",
+                    "mime_type": "media-mime-type",
+                    "sha256": "checksum",
+                    "caption": "image-caption",
+                },
+                "location": {
+                    "address": "1 Hacker Way, Menlo Park, CA, 94025",
+                    "name": "location-name",
+                },
+                "system": {"body": "system-message-content"},
+                "text": {"body": "text-message-content"},
+            },
+        ),
+        self.assertEqual(messages.created_by, user.username)
+        self.assertEqual(messages.recipient_type, "type")
```

Comment:
I'm not sure what the `recipient_type` field is, but there's no mention of it in the WhatsApp documentation. 

Maybe we can remove this field from the model?

Reply:
okay, removed